prometheus_flask_exporter/__init__.py

```diff
@@ -413,24 +421,32 @@ def teardown_request(exception=None):
 
             response = make_response('Exception: %s' % exception, 500)
 
+            if callable(duration_group):
+                group = duration_group(request)
+            else:
+                group = getattr(request, duration_group)
+
+            request_exceptions_labels = {
+                'method': request.method,
+                'status': 500,
```

Comment:
Not sure if this should be hardcoded or inferred from the kind of exception.
Also, if status is always 500, why have it as a label at all?

Reply:
Ah, yes - I think I got confused with the hard-coded 500 during the histogram update on master. let me clean this up, I'll ping you once that's done.